src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +13,17 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
+
+        UserService userService = new UserService();
```

Comment:
You create new user service instance each time method is called, it's inefficient, let's create on per class, move it to a class level constant, private final